README.md

```diff
@@ -91,18 +91,30 @@ Collect and process real time twitter data to analyse popularity of tweets with
 - nltk
 - networkx
 - streamlit
-- pandas,numpy
+- pandas
+- numpy
 - matplotlib
 
 <br>
 </br>
 
 ## Setup Locally
 
-- Go the the cloned directory on your local machine.
+- Clone the repository on your local machine.
+  ```sh
+  git clone https://github.com/kaustav202/RealTime-TwitterDataAnalysis.git
+  ```
+- Go into the cloned directory
 - Run `pip install -r requirements.txt` to install all the dependencies.
+- Create a developer account on twitter: https://developer.twitter.com/en
 - Get your Twitter API credentials and replace the placeholders in twitter_config.py.
-- run `python main.py` which is the application entry point.
+    - Go to the Twitter Developer Portal Projects & Apps page at https://developer.twitter.com/en/portal/projects-and-apps
+    - Find the API/consumer key and secret under the Consumer Keys section of the Keys and Tokens tab of your app
+    - Your account's access token and secret for your app can be found under the Authentication Tokens section of the Keys and Tokens tab of your app
+- From inside the `app/` folder, you can run `python stream.py` which adds(streams) the tweets into `tweets.json`
+- Run `python main.py` which is the application entry point preferably after some time so that you have more tweets to perform the analysis.
+- You can also perform Sentiment Analysis by running `python sentiment_analysis.py` and draw tweet network graphs by running `python tweet_network.py`
+- Remember that the streaming of tweets is a completely different and independent step than running the analyses
```

Comment:
Mention it needs to be done initially and just remove different .. (completely independent step) is fine
```suggestion
- Remember that the streaming (writing) of tweets is a completely independent step that needs to be performed initially by running stream.py
```